eor_limits/process_munoz_2021.py

```diff
@@ -12,7 +12,7 @@
 from eor_limits.data import DATA_PATH
 
 
-def get_munoz_2021_line(model="EOS", redshift=None, linewidth=1.0):
+def get_munoz_2021_line(model="EOS", redshift=None, linewidth=1.0, nf=None):
```

Comment:
If needed, this should be in the docstring. But it doesn't look like this is actually used anywhere?

Reply:
Correct, not sure what I intended for that...